screens/Scan/ScanLayoutController.ts

```diff
@@ -203,13 +200,13 @@ export function useScanLayout() {
 
   useEffect(() => {
     if (isDone) {
-      navigation.navigate('home', { activeTab: 0 });
+      navigation.navigate(BOTTOM_TAB_ROUTES.home(0).name);
```

Comment:
can you please check, if activeTab plays any role here, if not please remove it from the routeConstants file as well.

Reply:
done i have removed it from routeConstants as it not being used. Before New UI merge we have 3 different tabs in home screen and the active tab variable is being used to display the current active tab and now we are maintaining different screens for all the tabs so activeTab is not required.

we can remove it from the other files as part of clean up task